pkg/exec/helpers.go

```diff
@@ -62,8 +62,8 @@ func RunErrorForError(err error) *RunError {
 // stderr + stdout, it scans these for lines and returns a slice of output lines
 func CombinedOutputLines(cmd Cmd) (lines []string, err error) {
 	var buff bytes.Buffer
-	cmd.SetStdout(&buff)
-	cmd.SetStderr(&buff)
+	cmd = cmd.SetStdout(&buff)
```

Comment:
just curious, we need the assignment to make the code testable with the new `fakeCmder`?

Reply:
At one point we did. This is more correct regardless.